api-description/forms/pk/post.json

```diff
@@ -2,22 +2,35 @@
     "description": "Endpoint to answer a form",
     "url": "http://localhost:8000/forms/1",
     "fields": {
-        "answers": {
-            "1": "Friends",
-            "2": "one_time",
-            "3": ["two_times", "three_times"],
-            "4": 5
-        }
+        "user_responses": [
+            {
+                "fieldset": 1,
+                "user_id": "User#20#",
```

Comment:
What we can do is to remove it from the post request and save the user in the view, something like what we did in [`humira`](https://github.com/incuna/abbvie-humira-epatient/blob/5318d57eb2ad5c5409960f7f5a91001d4198f144/api/tracker/mixins.py#L31).

```python
    def perform_create(self, serializer):
        serializer.save(patient=self.get_patient())
```